zap/src/main/java/org/parosproxy/paros/model/Session.java

```diff
@@ -1362,7 +1362,7 @@ public void deleteContext(Context c) {
 
     public Context getContext(int index) {
```

Comment:
The parameter could be renamed as well. There's also the variant `contextIndex`.

Reply:
I'll tackle that shortly, as well as adding a ZAP comment to all the modified paros files.

Reply:
Thanks.